src/kafka.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const {KafkaProducer, KafkaConsumer} = require('./kafkaUtils')
+const {port} = require('./config').getConfig()
+const logger = require('./logger')
+
+let kafkaConsumer, kafkaProducer
+
+const initiateKafkaClient = async (endpoints = []) => {
+  kafkaConsumer = new KafkaConsumer()
+  kafkaProducer = new KafkaProducer()
+
+  for (let i = 0; i < endpoints.length; i++) {
+    const endpoint = endpoints[i]
+
+    await subscribeTopicToConsumer(endpoint)
+  }
+}
+
+const subscribeTopicToConsumer = async endpoint => {
+  if (endpoint.kafkaConsumerTopic) {
+    await kafkaConsumer.subscribe(endpoint.kafkaConsumerTopic, {
+      url: `http://localhost:${port}/${endpoint.endpoint.pattern}`,
```

Comment:
```suggestion
      url: `http://localhost:${port}${endpoint.endpoint.pattern}`,
```
Since `endpoint.endpoint.pattern` follows the pattern: "/pattern" we should remove the / otherwise we end up with //

Reply:
Addressed in https://github.com/jembi/openhim-mediator-mapping/pull/205/commits/55e9fa320987315eb714daea597f6e93775da7c0